static/app/views/replays/detail/eventMetaData.tsx

```diff
@@ -19,20 +19,17 @@ type Props = {
 };
 
 function EventMetaData({crumbs, durationMs, replayRecord}: Props) {
-  const {projects} = useProjects();
+  const {
+    params: {replaySlug},
+  } = useRouteContext();
+  const [slug, id] = replaySlug.split(':');
```

Comment:
I suppose this works - there's always the possibility of the project slug changing, but I don't see that being a concern for us.

Reply:
`useReplayData` uses the `projectSlug` straight out of the browsers-url to fetch all the data, just like it uses `orgSlug`. So i think if the slugs change the user will need to start over and navigate to the new url.

example, from useReplayData():
```typescript
  const [projectSlug, replayId] = replaySlug.split(':');
  ...
  const fetchReplay = useCallback(async () => {
    const response = await api.requestPromise(
      `/projects/${orgSlug}/${projectSlug}/replays/${replayId}/`
    );
    return response.data;
  }, [api, orgSlug, projectSlug, replayId]);
```